scripts/build-packages-and-demos.sh

```diff
@@ -1,35 +1,36 @@
-# Generate all assets
-# needed for push to gh-page
+# README:
+# each package should build a dist folder that contains everything needed to publish that package
+# including:
+#  - package.json
+#  - .js files and .d.ts files (no raw .ts files!)
+#  - README.md
+#  - and any specialty files
+
+# Generate all assets needed for push to gh-pages
 mkdir pages
 touch pages/.nojekyll
 # echo "charts.carbondesignsystem.com" > pages/CNAME
 
-# Build Core demos and copy to `pages` directory
+# Core build
 cd packages/core
 npm run build
 npm run demo:build
 typedoc --out ./demo/bundle/documentation ./src/index.ts
 cp -a demo/bundle/. ../../pages
 
-# Build Angular demos and copy to `pages` directory
+# Angular build
 cd ../angular
-# Build angular demos
+npm run build
 npm run build-storybook
 cp -a storybook-dist/. ../../pages/angular
-# Build angular bundle for release
-npm run build
-mv dist ../charts-angular-dist
-cd ..
-rm -rf angular
-mv charts-angular-dist angular
 
-# Build React demos
-cd react
+# React build
+cd ../react
 npm run build
 npm run build-storybook
 cp -a storybook-dist/. ../../pages/react
 
-# Build Vue demos
+# Vue build
 cd ../vue
 npm run build
```

Comment:
If we're going to have a `build` script in every package, then we can run `lerna run build` and it'll build all, and so on for the other standard commands as well

Reply:
That would definitely simplify things

Reply:
👍

Reply:
The demo build is `build-storybook` and `demo:build` ... if we standardize on one (`demo:build` I think) that'll work too

Reply:
Each demo should be able to hoist itself to the top level pages directory too 🤔

Reply:
for sure, this whole file can be shrunk to the a few lines

Reply:
Or that can just be a separate collect step in the deploy script ... that's probably cleaner

Reply:
Pretty sure using `lerna run` you can have a script in root run in every package directory, that'll just regex all these folders and copy them over to `root/pages`

Reply:
Yep! `lerna exec` + some mild demo dir changes and it works super well 🎉